plugins/examples/pod-counter/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "allowSyntheticDefaultImports": true,
+    "jsx": "react-jsx",
+    "moduleResolution": "node",
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "node_modules/*.js",
+        "node_modules/*/index.js",
+        "node_modules/@kinvolk/headlamp-plugin/node_modules/*",
+        "node_modules/@kinvolk/headlamp-plugin/node_modules/*/index.js"
+      ],
+      "@iconify/react": [
+        "node_modules/@kinvolk/headlamp-plugin/node_modules/@iconify/react/src/icon.js"
+      ],
+      "@kinvolk/headlamp-plugin/lib/K8s": [
+        "node_modules/@kinvolk/headlamp-plugin/types/src/lib/k8s/index.d.ts"
+      ],
+      "@kinvolk/headlamp-plugin/lib/CommonComponents/*": [
+        "node_modules/@kinvolk/headlamp-plugin/types/src/components/common/*/index.d.ts",
+        "node_modules/@kinvolk/headlamp-plugin/types/src/components/common/*.d.ts"
+      ]
+    },
+    "rootDirs": ["node_modules/@kinvolk/headlamp-plugin/"],
+    "skipLibCheck": true
+  },
+  "include": ["./src/**/*"]
+}
```

Comment:
Unrelated to your PR, but I am wondering now whether we could use a hardlink for this, and ship the tsconfig in the headlamp-plugin too, so it'd always be updated...

Reply:
Yeah, would definitely be good to have. I guess it is possible (other tools do something like this).

Reply:
Added an issue for this: https://github.com/kinvolk/headlamp/issues/433